src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs

```diff
@@ -8,7 +8,7 @@
 // associated with an assembly.
 [assembly: AssemblyCompany("Autodesk, Inc")]
 [assembly: AssemblyProduct("Dynamo")]
-[assembly: AssemblyCopyright("Copyright � Autodesk, Inc 2023")]
+[assembly: AssemblyCopyright("Copyright � Autodesk, Inc 2023")]
```

Comment:
oh maybe our comuters do not have the same character set, but this should not be a real issue

Reply:
Yes, I have not figured out why my computer does this. Bummer... But not a real issue.